### ISSUE TITLE
Some services not shown in Icingaweb2 2.7.1 when grafana module enabled

### ISSUE THREAD

**Issue by manfredw** [Mikesch-mp/icingaweb2-module-grafana#221]
Some services not shown in Icingaweb2 2.7.1 when grafana module enabled.

Expected Behavior
Service with graph is shown.

## Current Behavior
Icingaweb2 shows error message `Undefined variable: dataSource` instead of service view.

Steps to Reproduce (for bugs)
Error rises with upgrade from Icingaweb2 2.6.x to 2.7.1.
When disabling grafana module the error is gone - service is shown.
Not all services are affected, services with spaces or slashes seems to be the problem.

Environment
icinga2 2.10.6
icingaweb2 2.7.1
grafana module 1.3.5
php 7.2.5

**Comment by Mikesch-mp** [comment]
Can you post your configuration.ini please and if you have cofigured graphs also the graphs.ini

**Comment by manfredw** [comment]
config.ini (censored some data):
```
[grafana]
version = "1"
host = "X.X.X.X:3000"
protocol = "https"
timerange = "2d"
timerangeAll = "1w/w"
defaultdashboard = "Icingaweb2"
defaultdashboarduid = "XXXXXXX"
defaultdashboardpanelid = "2"
defaultorgid = "1"
shadows = "0"
theme = "light"
datasource = "influxdb"
accessmode = "indirectproxy"
height = "280"
width = "640"
enableLink = "yes"
debug = "0"
ssl_verifypeer = "0"
ssl_verifyhost = "0"
proxytimeout = "5"
authentication = "token"
apitoken = "XXXXXXX=="
indirectproxyrefresh = "yes"
usepublic = "no"
[hostalive]
dashboard = "Icingaweb2"
panelId = "2"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"
```

**Comment by manfredw** [comment]
graphs.ini (also censored some data)
Graph name sometimes reference to check command, in other cases to service name (i.e. Disk, because it uses different commands for different OS) - worked up to icingaweb 2.6.x.

```
[vmware-host-cpu-usage]
dashboard = "Icingaweb2"
panelId = "5"
orgId = "1"
timerange = "24h"
repeatable = "no"
dashboarduid = "XXXX"

[vmware-host-mem-usage]
dashboard = "Icingaweb2"
panelId = "6"
orgId = "1"
timerange = "24h"
repeatable = "no"
dashboarduid = "XXXX"

[vmware-host-volumes]
dashboard = "Icingaweb2"
panelId = "4"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"

[vmware-host-io-read-latency]
dashboard = "Icingaweb2"
panelId = "8"
orgId = "1"
timerange = "24h"
repeatable = "no"
dashboarduid = "XXXX"

[vmware-host-io-write-latency]
dashboard = "Icingaweb2"
panelId = "8"
orgId = "1"
timerange = "24h"
repeatable = "no"
dashboarduid = "XXXX"

[ntp-peer]
dashboard = "Icingaweb2"
panelId = "12"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"

[snmp-memory]
dashboard = "Icingaweb2"
panelId = "7"
orgId = "1"
timerange = "24h"
repeatable = "no"
dashboarduid = "XXXX"

[Phones registered]
dashboard = "Icingaweb2"
panelId = "14"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"

[Disk]
dashboard = "Icingaweb2"
panelId = "4"
orgId = "1"
customVars = "&var-storagename=$snmp_storage_name$"
repeatable = "no"
dashboarduid = "XXXX"

[NTP]
dashboard = "Icingaweb2"
panelId = "12"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"

[CPU Usage]
dashboard = "Icingaweb2"
panelId = "5"
orgId = "1"
timerange = "24h"
repeatable = "no"
dashboarduid = "XXXX"

[Devices Registered]
dashboard = "Icingaweb2"
panelId = "14"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"

[tcp]
dashboard = "Icingaweb2"
panelId = "16"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"

[vmware-host-runtime-temp]
dashboard = "Icingaweb2"
panelId = "18"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"

[Icinga Status]
dashboard = "icinga-system-status"
panelId = "1,17,2,18,11"
orgId = "1"
repeatable = "no"
dashboarduid = "XXXX"
```

**Comment by Mikesch-mp** [comment]
Is fixed in 1.3.6.
